### PR TITLE
chore(data): sync

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Facing error when I run `validate-all` task.
opened this to get inputs from team

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/44"></a>

### CODE REVIEW

**Comment by yasserfaraazkhan** [comment]
Hi @saturninoabril , 
I couldn't understand this error, can you help me by pointing which attribute is not set rightly

**Comment by saturninoabril** [comment]
I think it's a bug. Currently, folder creation only supports whenever the parent is present or there is new test case that's introducing such new parent folder. In this case, "Self-Hosted" does not exist in Zephyr and no new test case is a direct child of such folder. The new test cases were under the "In product purchase" (like grandchild). Maybe good as feature enhancement but would be low priority in my opinion.

The workaround I did were directly in Zephyr UI:
- Created "Self-Hosted".
- Deleted multiple duplicates of "In product purchase" folders.
- Other
  - Instead of "Follow steps in <test-1.md>", used "Call To Test" to "MM-T5367" feature in Zephyr UI to not repeat prerequisites. Note that this repo supports rendering of "Call To Test" but not saving. So, it needs to be made in Zephyr UI currently.

File: .github/workflows/deno.yml [hunk 1]
```diff
@@ -26,3 +26,6 @@ jobs:
 
       - name: Validate changes
         run: deno task validate
+
+      - name: Validate All
+        run: deno task validate-all
```
Comment (saturninoabril) [inline .github/workflows/deno.yml:31]: In the future, I think this is good to have in separate PR. But I'm fine as is.

**Review by saturninoabril — APPROVED** [review]
Thanks @yasserfaraazkhan, LGTM.

**Comment by yasserfaraazkhan** [comment]
Thank you for quick fix Saturn.
I've noted the points for future tests.

**Pull request merged**